test/health_checker_unittest.cc

```diff
@@ -1,4 +1,5 @@
 #include "../src/health_checker.h"
+#include "../src/store.h"
```

Comment:
We're not testing the new functionality... Are you expecting to add tests in a later PR?

Reply:
Yes, I'll file a bug to do so.